docs/quick-start/index.md

```diff
@@ -0,0 +1,487 @@
+
+        return np.array(doc1.similarity(doc2))
+```
+
+Now that we have our model file ready to go, the last piece of the puzzle is to tell MLServer a bit of info 
+about it. In particular, it wants (or needs) to know the name of the model and how to implement 
+it. The former can be anything you want (and it will be part of the URL of your API), and the latter will 
+follow the recipe of `name_of_py_file_with_your_model.class_with_your_model`.
+
+Let's create the `model-settings.json` file MLServer is expecting inside our `similarity_model` directory 
+and add the name and the implementation of our model to it.
+
+
+```json
+# models_hub/similarity_model/model-settings.json
+
+{
+    "name": "doc-sim-model",
+    "implementation": "my_model.MyKulModel"
+}
+```
+
+    Writing models_hub/similarity_model/model-settings.json
+
+
+Now that everything is in place, we can start serving predictions locally to test how things would play 
+out for our future users. We'll initiate our server via the command line, and later on we'll see how to 
+do the same via Python files. Here's where we are at right now in the process of developing microservices 
+with MLServer.
+
+![start](../assets/start_service.png)
+
+As you can see in the image, our server will be initialized with three entry points, one for HTTP requests, 
+another for gRPC, and another for the metrics. To learn more about the powerful metrics feature of MLServer, 
+please visit the relevant docs page [here](https://mlserver.readthedocs.io/en/latest/user-guide/metrics.html). 
+To learn more about gRPC, please see this tutorial [here](https://realpython.com/python-microservices-grpc/).
+
+To start our service, open up a terminal and run the following command.
+
+```bash
+mlserver start models_hub/similarity_model/
+```
+
+Note: If this is a fresh terminal, make sure you activate your environment before you run the command above. 
+If you run the command above from your notebook (e.g. `!mlserver start models_hub/similarity_model/`), 
+you will have to send the request below from another notebook or terminal since the cell will continue to run 
+until you turn it off.
+
+## 04 Testing our Service
+
+Time to become a client of our service and test it. For this, we'll set up the payload we'll send 
+to our service and use the `requests` library to [POST](https://www.baeldung.com/cs/http-get-vs-post) our request.
+
+
+```python
+from mlserver.codecs import StringCodec
+import requests
+```
+
+Please note that the request below uses the variables we created earlier with the summaries 
+of Barbie and Oppenheimer. If you are sending this POST request from a fresh python file, make 
+sure you move those lines of code into your request file.
+
+
+```python
+inference_request = {
+    "inputs": [
+        StringCodec.encode_input(name='docs', payload=[barbie, oppenheimer], use_bytes=False).dict()
+    ]
+}
+inference_request
+```
+
+
+
+
+    {'inputs': [{'name': 'docs',
```

Comment:
You seem to have included the inference request twice? Once with variables and once with the raw text. I'd suggest just keeping the one with variables as you have already explained above that they'll need to be able to reference them from earlier